File: tests/data-clumps/test-cases/field-field/always-class/positive/java/source/Doctor.java
public class Doctor {
    public Address address;
    public ContactInfo contactInfo;
    public Insurance insurance;
    public String licenseNumber;

    public Doctor(Address address, ContactInfo contactInfo, Insurance insurance, String licenseNumber) {
        this.address = address;
        this.contactInfo = contactInfo;
        this.insurance = insurance;
        this.licenseNumber = licenseNumber;
    }
}

File: tests/data-clumps/test-cases/field-field/always-class/positive/typescript/source/Doctor.ts
import { Address } from './Address';
import { ContactInfo } from './ContactInfo';
import { Insurance } from './Insurance';

export class Doctor {
  public address: Address;
  public contactInfo: ContactInfo;
  public insurance: Insurance;
  public licenseNumber: string;

  constructor(
    address: Address,
    contactInfo: ContactInfo,
    insurance: Insurance,
    licenseNumber: string
  ) {
    this.address = address;
    this.contactInfo = contactInfo;
    this.insurance = insurance;
    this.licenseNumber = licenseNumber;
  }
}
